sycl/include/CL/cl_usm_ext.h

```diff
@@ -67,26 +67,26 @@ typedef cl_uint cl_unified_shared_memory_type_intel;
 typedef cl_uint cl_mem_advice_intel;
 
 /* cl_mem_advice_intel */
-#define CL_MEM_ADVICE_TBD0_INTEL 0x41A8 /* reserved for future */
-#define CL_MEM_ADVICE_TBD1_INTEL 0x41A9 /* reserved for future */
-#define CL_MEM_ADVICE_TBD2_INTEL 0x41AA /* reserved for future */
-#define CL_MEM_ADVICE_TBD3_INTEL 0x41AB /* reserved for future */
-#define CL_MEM_ADVICE_TBD4_INTEL 0x41AC /* reserved for future */
-#define CL_MEM_ADVICE_TBD5_INTEL 0x41AD /* reserved for future */
-#define CL_MEM_ADVICE_TBD6_INTEL 0x41AE /* reserved for future */
-#define CL_MEM_ADVICE_TBD7_INTEL 0x41AF /* reserved for future */
+#define CL_MEM_ADVICE_TBD0_INTEL 0x4208 /* reserved for future */
```

Comment:
How can I check if new values are valid?

Reply:
https://github.com/KhronosGroup/OpenCL-Docs/commit/308d6e49b44d453ace52fd8de72fe577a16f1b80

The initial values we were given overlapped with a Qualcomm range.  Unfortunately this took a few weeks to be discovered.